Sources/MapboxMaps/Snapshot/Snapshotter.swift

```diff
@@ -143,12 +137,16 @@ public class Snapshotter {
 
                 // Composite the logo on the snapshot,
                 // only after everything else has been drawn.
-                let logoView = LogoView(logoSize: .regular)
-                let logoPadding = CGFloat(10.0)
-                let logoOrigin = CGPoint(x: logoPadding,
-                                         y: uiImage.size.height - logoView.frame.size.height - logoPadding)
-                context.translateBy(x: logoOrigin.x, y: logoOrigin.y)
-                logoView.layer.render(in: context)
+                if self.options.showsLogo {
```

Comment:
Is capturing a strong reference to self here intentional? Previously, self was not captured in this block. What are the implications?